bigchaindb/web/views/outputs.py

```diff
@@ -24,5 +24,5 @@ def get(self):
         with pool() as bigchain:
             outputs = bigchain.get_outputs_filtered(args['public_key'],
                                                     include_spent)
-            # NOTE: We pass '..' as a path to create a valid relative URI
-            return [u.to_uri('..') for u in outputs]
```

Comment:
The [`to_uri` method](https://github.com/bigchaindb/bigchaindb/blob/master/bigchaindb/common/transaction.py#L196) might have only been used for this API endpoint.

As we're not using it here anymore, we might want to check if we cannot simply remove it now.